quipucords/quipucords/environment.py

```diff
@@ -113,31 +122,38 @@ def startup():
     logger.info("\n".join(env_list))
     logger.info("%s END ENVIRONMENT VARIABLES %s", mark, mark)
 
-    QPC_POSTGRES_DBMS = "postgres"
-    QPC_SQLITE_DBMS = "sqlite"
-    valid_dbms = [QPC_POSTGRES_DBMS, QPC_SQLITE_DBMS]
-    qpc_dbms = os.environ.get("QPC_DBMS")
-    if qpc_dbms in valid_dbms:
-        logger.info('QPC_DBMS set to "%s".', qpc_dbms)
-        if qpc_dbms == QPC_POSTGRES_DBMS:
-            database = os.getenv("QPC_DBMS_DATABASE", "postgres")
-            user = os.getenv("QPC_DBMS_USER", "postgres")
-            host = os.getenv("QPC_DBMS_HOST", "localhost" or "::")
-            # pylint: disable=invalid-envvar-default
-            port = os.getenv("QPC_DBMS_PORT", 5432)
-            logger.info('QPC_DBMS_HOST set to "%s"', host)
-            logger.info('QPC_DBMS_PORT set to "%s"', port)
-            logger.info('QPC_DBMS_DATABASE set to "%s"', database)
-            logger.info('QPC_DBMS_USER set to "%s"', user)
-    elif not qpc_dbms:
-        logger.info('QPC_DBMS not set. Using default of "postgres".')
-    else:
-        logger.info(
-            'QPC_DBMS was set to "%s" which is not a valid option. '
-            'Using default of "postgres".',
-            (qpc_dbms),
-        )
 
+def log_database_configuration():
+    """Log settings related to the database configuration."""
+    settings_to_log = {
+        "postgres": [
+            "QPC_DBMS",
+            "QPC_DBMS_HOST",
+            "QPC_DBMS_PORT",
+            "QPC_DBMS_DATABASE",
+            "QPC_DBMS_USER",
+        ],
+        "sqlite": [
+            "QPC_DBMS",
+            "DB_PATH",
+        ]
+    }
+
+    for setting_name in settings_to_log[settings.QPC_DBMS]:
+        logger.info("%s set to %s", setting_name, repr(getattr(settings, setting_name)))
```

Comment:
<3

Reply:
In a future refactoring step, this is also the pattern I would follow to log more/all of our settings. Start with lists or a dict categorized by general functional area, and then just iterate through them all.